cupyx/scipy/ndimage/_filters_core.py

```diff
@@ -0,0 +1,378 @@
+import warnings
+
+import cupy
+
+
+def _get_output(output, input, shape=None):
```

Comment:
I think we can use `zeros_like`instead of this function
we can override shape and dtype

https://docs.cupy.dev/en/stable/reference/generated/cupy.zeros_like.html

Reply:
Not exactly. This function, if given an array for `output` returns that array, not a newly allocated array. However, the `else` can probably be replaced with `cupy.zeros_like(input, dtype=output, shape=shape)`.